res/values/strings.xml

```diff
@@ -706,7 +706,7 @@
 
 
     <!-- system messages -->
-    <string name="systemmsg_group_name_changed">Group name changed from \"%1$s\" to \"%2$s\".</string>
+    <string name="systemmsg_group_name_changed">Name of group %1$s changed as \"%2$s\".</string>
```

Comment:
we should either leave the quotes or remove them; i think, the quotes make sense here. "changed as" does not look correct to me. also "Group name" sounds good to me.

Reply:
@r10s Yes the old string is better. This feels a bit haphazard like many changes in this PR.